department-of-veterans-affairs/va.gov-cms
# VACMS-1384: Update all instances of Revision Log Widget (hide_revision_field) to Textarea

## Description

See #1384 .

This PR changes form display widget for Revision Log Message to Textarea.

**NOTE:** Once this Pr is merged and deployed, I'll open a PR for removal of hide_revision_field contrib.
## Testing done

Manual

## Screenshots


## QA steps

As user 1
1. visit /admin/content/facility-status. -> select a few facilities and click on "Update Operating Status" button
   - [x]  Confirm that the VBO form for updating fields is displayed without errors
1. visit add Event node form /node/add/event
   - [x] Confirm that Editorial workflow fields match the screenshot below
<img width="749" alt="Screen Shot 2020-04-14 at 12 10 58 PM" src="https://user-images.githubusercontent.com/16477724/79258907-13e97200-7e49-11ea-9357-15273d3687f4.png">

1. visit several other random content type forms
   - [x] confirm that Editorial workflow fields match the screenshot above


## Definition of Done
